src/ribbon/buttonbar.cpp

```diff
@@ -130,15 +128,32 @@ class wxRibbonButtonBarButtonBase
 
         if ( bitmap_large.IsOk() )
         {
+            bitmap_large.SetScaleFactor(scale);
+            bitmap_large_disabled.SetScaleFactor(scale);
+
+            //Behavior of ImageList is not the same on different platforms
```

Comment:
We really need to isolate and fix this problem instead of working around it here.

Reply:
Done.